rapids_config.cmake

```diff
@@ -26,11 +26,14 @@ else()
   )
 endif()
 
-if(NOT EXISTS "${CMAKE_CURRENT_BINARY_DIR}/CUDF_RAPIDS-${RAPIDS_VERSION_MAJOR_MINOR}.cmake")
-  file(
-    DOWNLOAD
-    "https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-${RAPIDS_VERSION_MAJOR_MINOR}/RAPIDS.cmake"
-    "${CMAKE_CURRENT_BINARY_DIR}/CUDF_RAPIDS-${RAPIDS_VERSION_MAJOR_MINOR}.cmake"
-  )
-endif()
+include(FetchContent)
+# GIT_TAG = last commit prior to https://github.com/rapidsai/rapids-cmake/pull/678
+FetchContent_Declare(
+  rapids-cmake
+  GIT_REPOSITORY https://raw.githubusercontent.com/rapidsai/rapids-cmake.git
+  GIT_TAG        0e65a8481b9895b5075d2b601690540a21b45deb
+)
+file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-24.10/RAPIDS.cmake
+  ${CMAKE_CURRENT_BINARY_DIR}/CUDF_RAPIDS-${RAPIDS_VERSION_MAJOR_MINOR}.cmake)
+
```

Comment:
```suggestion
set(rapids-cmake-sha 0e65a8481b9895b5075d2b601690540a21b45deb)
if(NOT EXISTS "${CMAKE_CURRENT_BINARY_DIR}/CUDF_RAPIDS-${RAPIDS_VERSION_MAJOR_MINOR}.cmake")
  file(
    DOWNLOAD
    "https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-${RAPIDS_VERSION_MAJOR_MINOR}/RAPIDS.cmake"
    "${CMAKE_CURRENT_BINARY_DIR}/CUDF_RAPIDS-${RAPIDS_VERSION_MAJOR_MINOR}.cmake"
  )
endif()
```

Reply:
Yeah this is way better, totally agree. Thanks for updating the outdated doc I'd been referencing (https://github.com/rapidsai/rapids-cmake/pull/681).

Now that the fix for this is in (https://github.com/rapidsai/rapids-cmake/pull/679), I won't apply this change... but good to know for next time.